src/interface/src/app/treatments/map-base-layer/map-base-layer.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { NgForOf } from '@angular/common';
+import {
+  baseLayerStyles,
+  BaseLayerType,
+  DEFAULT_BASE_MAP,
+} from '../treatment-map/map-base-layers';
+import { MapConfigState } from '../treatment-map/map-config.state';
+
+@Component({
+  selector: 'app-map-base-layer',
+  standalone: true,
+  imports: [NgForOf],
+  templateUrl: './map-base-layer.component.html',
+  styleUrl: './map-base-layer.component.scss',
+})
+export class MapBaseLayerComponent {
```

Comment:
Another detail, in the next pr im renaming this to baseLayerControl - to differentiate this components with map layer ones